modules/tensor_mechanics/src/materials/ComputeReturnMappingStress.C

```diff
@@ -37,9 +37,9 @@ void
 ComputeReturnMappingStress::initialSetup()
 {
   std::vector<MaterialName> models = getParam<std::vector<MaterialName> >("return_mapping_models");
-  for (unsigned i = 0; i < models.size(); ++i)
+  for (unsigned int i = 0; i < models.size(); ++i)
   {
-    MooseSharedPointer<RecomputeRadialReturn> rrr = MooseSharedNamespace::dynamic_pointer_cast<RecomputeRadialReturn>(getMaterialSharedPointerByName(models[i]));
+    StressUpdateBase * rrr = dynamic_cast<StressUpdateBase *>(&getMaterialByName(models[i]));
     if (rrr)
```

Comment:
@permcody @dschwen I'm not sure at all that this approach is the correct way to move to raw/dump pointers, but it seems to compile and pass the tests on my machine.  Please let me know if there's a better way to implement the pointers